doc/userdoc/citing-nest.rst

```diff
@@ -3,31 +3,16 @@
 Cite NEST
 =========
 
-Please cite NEST if you use it in your work. You can :doc:`let us know <community>` about your publications that used NEST, and we
+Please cite the version of NEST you used in your work. You can :doc:`let us know <community>` about your publications that used NEST, and we
 will add them to our `publication list <https://www.nest-simulator.org/publications/>`_; this will help make them
 visible to potential readers.
 
-For the full citation and additional formats follow the doi Zenodo link.
+For all versions from 2.8 onwards, you can find the full citation `on Zenondo <https://zenodo.org/search?page=1&size=20&q=nest-simulator>`_
```

Comment:
The link shows NEST and a lot of else. Is there any way to make it more specific?
This link 

```suggestion
For all versions from 2.8 onwards, you can find the full citation `on Zenondo <https://zenodo.org/search?page=1&size=20&q=nest-simulator&type=software>`_
```
brings it down to 336 instead of 5000+ items. Can that be improved further?

Reply:
@heplesser I couldn't find a really good result that limited it to only our software releases, I'll commit your suggestion.

Reply:
It's possible to limit it further by specifying the title and adding people from the creator list, for example:
https://zenodo.org/search?page=1&size=20&q=title:NEST%20AND%20creators.name:(Plesser,%20Hans%20Ekkehard)&type=software

Reply:
@hakonsbm thanks for the tip. I tried some other combinations, but that seems to be one of the best. If we don't want to include the name in there, this also worked https://zenodo.org/search?page=1&size=20&q=title:NEST%20AND%20filetype:gz%20-description:graphical&file_type=gz
@heplesser do you have a preference?

Reply:
I think we should stick to facts rather than people, so I prefer the approach linked to file type. I have removed one duplicity from your link and added sorting in descending order by publication date. Unfortunately, one cannot sort by title, and we have used the "version" field inconsistently over time (not the nest version), so we can't sort by that. My suggestion then is the following link:
https://zenodo.org/search?page=1&size=20&q=title:NEST%20AND%20-description:graphical&file_type=gz&sort=-publication_date